<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSVPA_Tab12</class>
 <widget class="QWidget" name="MSVPA_Tab12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current configuration:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="MSVPA_Tab12_CurrentConfigTE"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_45">
       <item>
        <widget class="QPushButton" name="MSVPA_Tab12_PrevPB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Go to previous tab page</string>
         </property>
         <property name="statusTip">
          <string>Go to previous tab page</string>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="MSVPA_Tab12_RunPB">
         <property name="toolTip">
          <string>Run the current MSVPA configuration</string>
         </property>
         <property name="statusTip">
          <string>Run the current MSVPA configuration</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_27">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
